src/schrodinger/rdkit_extensions/stereochemistry.cpp

```diff
@@ -154,6 +176,15 @@ void wedgeMolBonds(RDKit::ROMol& mol, const RDKit::Conformer* conf)
     for (auto bond : attachment_dummy_bonds) {
         bond->setBondType(RDKit::Bond::SINGLE);
     }
+
+    // Restore wiggly bond directions that were cleared by
+    // ClearSingleBondDirFlags
+    for (size_t bond_idx = 0; bond_idx < mol.getNumBonds(); ++bond_idx) {
+        if (had_wiggly_bond[bond_idx]) {
+            auto bond = mol.getBondWithIdx(bond_idx);
+            bond->setBondDir(RDKit::Bond::BondDir::UNKNOWN);
+        }
+    }
```

Comment:
@ricrogz @cdvonbargen -- does this also address the issue Ricardo described in this case? https://schrodinger.atlassian.net/browse/SKETCH-2584